app/javascript/controllers/toggle_field_controller.js

```diff
@@ -29,4 +29,8 @@ export default class extends Controller {
       }
     })
   }
+
+  toggleDisplay() {
```

Comment:
i guess this might not be necessary if i wrapped the Auth Tokens section in a turbo frame? i haven't checked yet.

Reply:
hmm, turbo doesn't quite work because the render is still based on `feed.privacy?`, which doesn't change until submit is clicked. so this method is used to toggle the display without relying on save/re-render.

Reply:
Ah, missed that.  Yeah, works for me!